### ISSUE TITLE
YOLOV3 help - "The testing results of the whole dataset is empty" 

### ISSUE THREAD

**Issue by mtarun02** [open-mmlab/mmdetection#5683]
Hello all,

I know my dataset is correct as it works with every model, except YOLOV3, where I get this error in between epochs and no val results while training. Below is my config file, could anyone point to any errors in model, etc. that could be leading to this issue?

I have left most of the default stuff the same, but cannot find any resources on what I might need to change for my dataset. 

Thank you.

![Screen Shot 2021-07-24 at 10 36 48 PM](https://user-images.githubusercontent.com/24819296/126885796-fc5b16ef-38a1-4158-8117-32776e6cd8b8.png)
![Screen Shot 2021-07-24 at 10 37 30 PM](https://user-images.githubusercontent.com/24819296/126885808-14334ab2-e0e9-41f5-b608-9bd065064432.png)
![Screen Shot 2021-07-24 at 10 37 48 PM](https://user-images.githubusercontent.com/24819296/126885816-27374493-63ed-471e-af1a-834356dad8f3.png)
![Screen Shot 2021-07-24 at 10 38 20 PM](https://user-images.githubusercontent.com/24819296/126885827-08fe545c-cfe0-4248-b352-47b96ed61710.png)

**Comment by mtarun02** [comment]
If anyone else has this error, fix by editing learning rate.

**Comment by Gavin-Hanks** [comment]
我通过更改学习率得到了结果。你可以试一下，但是带来的后果是训练很慢。
